test_snakes_cafe.py

```diff
@@ -77,3 +77,18 @@ def test_filled_sub_total():
     snakes_cafe.menus['Appetizers'][('Wings', 2.00)] = 5
     assert snakes_cafe.sub_total() == 10.00
     snakes_cafe.menus['Appetizers'][('Wings', 2.00)] = 0
+
+
+def test_generate_menu():
```

Comment:
Make sure you have docstrings for your tests as well. There should be a docstring on every function that you write.